ha/setup/create_pacemaker_resources.py

```diff
@@ -145,9 +151,8 @@ def s3servers(cib_xml, push=False, **kwargs):
         instance = int(kwargs["s3_instances"])
     except Exception as e:
         raise CreateResourceConfigError(f"Invalid s3 instance. Error: {e}")
-    for i in range(1, int(instance)+1):
-        process.run_cmd(f"pcs -f {cib_xml} resource create s3server-{i} ocf:seagate:dynamic_fid_service_ra \
-            service=s3server fid_service_name=s3server update_attrib=true \
+    for i in range(1, int(instance) + 1):
+        process.run_cmd(f"pcs -f {cib_xml} resource create s3server-{i} ocf:seagate:dynamic_fid_service_ra service=s3server fid_service_name=s3server \
```

Comment:
"update_attrib=true" is this expected change here?

Reply:
change was came through rebase, so removed it.